### ISSUE TITLE
Disciplines should be defined in a gistx namespace

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gistReferenceData#34]
Since `gist:Discipline` is a subclass of `gist:Category`, following the conventions we typically apply in client projects, these should go in a `gistx:` namespace, where 

```
@prefix gistx: <https://w3id.org/semanticarts/ns/taxonomy/gist/> .
```

We have not used this namespace yet, so a new redirect on the w3id site will be needed (@Jamie-SA can do this).

**Comment by uscholdm** [comment]
I don't think we are proposing to include instances of disciplines in gist, are we?   Not our usual policy. So, there is no need to create a gistx namespace, in the usual case. I am not a fan of adding discipline instances to gist ref data.

**Comment by rjyounes** [comment]
No, not to gist. They are already in the reference data. I believe they are useful in helping users pull out the aspects and units relevant to their domain.

**Comment by uscholdm** [comment]
Oh - right. Probably they should be in a gistx namespace.   I forget what we did for media types, probably they should also be in gistx?

**Comment by rjyounes** [comment]
Media types use these namespaces:

```
@prefix media-app: <https://www.iana.org/assignments/media-types/application/> .
@prefix media-txt: <https://www.iana.org/assignments/media-types/text/> .
```
